lightspeed/modules/proc_create-connection-secrets.adoc

```diff
@@ -99,12 +99,12 @@ This key controls whether the SSL certificate of the {ibmwatsonxcodeassistant} m
 Default = `true`
 
 |`model_enable_anonymization`
-d|_Optional, and supported on {PlatformName} {PlatformVers} only_
+d|_Optional, and supported on {PlatformName} version 2.5.250702 and later_
 
-This key controls whether the anonymization of Personally Identifiable Information (PII) is enabled. PII includes passwords, IP addresses, email addresses, and other sensitive data.
+This key controls whether the anonymization of Personally Identifiable Information (PII) is enabled. PII includes passwords, IP addresses, email addresses, and other sensitive data. When PII anonymization is enabled, users' personal information is modified to some generic values to protect their data and reduce the risk of data leaks. 
```

Comment:
suggest adding "the", so changin "When PII anonymization is enabled, users' personal information" > When PII anonymization is enabled, the users' personal information"

Reply:
Done